src/Http/Requests/SearchMediaRequest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace RonasIT\Media\Http\Requests;
+
+use RonasIT\Media\Contracts\Requests\SearchMediaRequestContract;
+use RonasIT\Support\BaseRequest;
+
+class SearchMediaRequest extends BaseRequest implements SearchMediaRequestContract
+{
+    public function rules(): array
+    {
+        return [
+            'page' => 'integer',
+            'per_page' => 'integer',
+            'all' => 'integer',
+            'query' => 'string',
+            'order_by' => 'string',
```

Comment:
please add `in` validation rule with available ordering fields

Reply:
Done.